### ISSUE TITLE
Subset of lakes with v. low DOC concentrations

### ISSUE THREAD

**Issue by diatomdaniel** [swulfing/STOICH.Aim1#6]
JC suggested that lakes in our data set close to the Great Lakes region w. very low DOC concentrations may be unit issues. We need to extract these data and check whether we have made any errors with our unit conversions.

**Comment by diatomdaniel** [comment]
Nice work @dnguyen2017. I guess we will discuss this on Friday...

**Comment by mcstreamy** [comment]
Yes well done @dnguyen2017 - we will discuss. Were other parameters out of wack in those LAGOS sample dates? PIs might have some insight on why DOC could be that low or issues with LAGOS dataset consistency (which I've heard mentioned before by Sarah)

**Comment by LinneaRock** [comment]
Thanks @dnguyen2017 et al.!! I took a look back through the workflows, original LAGOS datasets, and our compiled datasets. I believe those data are as reported in the LAGOS database.
